src/Popover/elements.js

```diff
@@ -1,34 +1,113 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-export const PopOver = styled.div`
-position: relative;
+export const ARROW_SIZE = 8;
+export const CONTENT_PADDING = 1.8;
+export const MARGIN = 12;
 
-width: ${({ width }) => width};
+export const Arrow = styled.span`
+  color: ${({ theme: { palette } }) => palette.white};
 
-padding: 1.8rem;
+  text-shadow: ${({ placement }) =>
```

Comment:
It's a bit redundant, move this part of code directly the right place, where you check for placement and add css